utils/releaser/src/ReleaseWorker/AfterRelease/CheckHeimdallBuildReleaseWorker.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker\AfterRelease;
+
+use PharIo\Version\Version;
+use Shopsys\Releaser\ReleaseWorker\AbstractShopsysReleaseWorker;
+use Shopsys\Releaser\Stage;
+
+final class CheckHeimdallBuildReleaseWorker extends AbstractShopsysReleaseWorker
+{
+    /**
+     * Higher first
+     * @return int
+     */
+    public function getPriority(): int
+    {
+        return 260;
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     * @return string
+     */
+    public function getDescription(Version $version): string
+    {
+        return 'Check builds on heimdall';
+    }
+
+    /**
+     * @return string
+     */
+    public function getStage(): string
+    {
+        return Stage::AFTER_RELEASE;
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     */
+    public function work(Version $version): void
+    {
+        $this->confirm('Confirm heimdall build passes');
```

Comment:
```suggestion
        $this->symfonyStyle->note('It is suitable to discard the deletion of the application in Kubernetes for the new tag');
        $this->symfonyStyle->note('Currently, master will fail anyway because in microservice, there is a dependency on coding standards that are not released yet at the given moment.')
        $this->confirm('Confirm heimdall build passes');
```